api/views.py

```diff
@@ -93,4 +92,10 @@ def change_seat_status(request, pk):
     serialseats = ChangeReservationSeatStatusSerializer(seats, many=False)
     if not seats.is_occupied:
         seats.is_occupied = True
-    return Response(serialseats.data)
\ No newline at end of file
+    return Response(serialseats.data)
+
+
+class PossibleFraudsReservations(viewsets.ReadOnlyModelViewSet):
```

Comment:
We should usually add the ViewSet in the class definition, even though we don't have to.